client/src/components/layout/TopBar.js

```diff
@@ -24,10 +24,13 @@ const TopBar = ({ user }) => {
     <div className="top-bar">
       <div className="top-bar-left">
         <ul className="menu">
-          <li className="menu-text">App</li>
+          <li className="menu-text">Triper</li>
           <li>
             <Link to="/">Home</Link>
           </li>
+          <li>
+            <Link to="/">Add a Trip</Link>
```

Comment:
Adding this link here should ideally be done when you're on the New Trip feature. If this is merged into your `main` branch and a user sees this, it's confusing since it brings them to a page where they can't add a trip because that hasn't been built yet